.github/workflows/build.yaml

```diff
@@ -111,13 +111,13 @@ jobs:
 
       # Note that this approach to cloning repos obviates any modification to
       # the CMake variables in get_cumlprims_mg.cmake since CMake will just use
-      # the clone as is.
+      # the clone as-is.
       extra-repo: rapidsai/cugraph-ops
-      extra-repo-sha: branch-23.08
+      extra-repo-sha: branch-23.06
       extra-repo-deploy-key: CUGRAPH_OPS_SSH_PRIVATE_DEPLOY_KEY
 
-      before-wheel: "RAPIDS_PY_WHEEL_NAME=pylibcugraph_${{ '${PIP_CU_VERSION}' }} rapids-download-wheels-from-s3 /local-wheelhouse"
-      skbuild-configure-options: "-DDETECT_CONDA_ENV=OFF -DCUGRAPH_BUILD_WHEELS=ON -DFIND_CUGRAPH_CPP=OFF -DCPM_cugraph-ops_SOURCE=${GITHUB_WORKSPACE}/python/cugraph/cugraph-ops/"
+      skbuild-configure-options: "-DDETECT_CONDA_ENV=OFF -DCUGRAPH_BUILD_WHEELS=ON -DFIND_CUGRAPH_CPP=OFF -DCPM_cugraph-ops_SOURCE=/project/cugraph-ops/"
+      uses-setup-env-vars: false
```

Comment:
This looks off, but it is likely due to some issue with the branch merged (as noted above)

Reply:
I believe Vyas resolved this in [a6d0cb0](https://github.com/rapidsai/cugraph/pull/3456/commits/a6d0cb0e3785d96a548785181b728daa4c141952)